p2p/ipld/README.md

```diff
@@ -0,0 +1,18 @@
+# LazyLedger ipld-plugin
+
+## Installation
+
+Usually, a simple `make install` should build and install the plugin locally.
+
+If you want to compile against a locally checked out version of IPFS, run:
+
+```sh
+cd plugin
+./set-target.sh REPLACE_WITH_YOUR_LOCAL_PATH/go-ipfs
+make install
+```
```

Comment:
I actually had to do this (` ./set-target.sh ~/go-projects/go-ipfs`). Otherwise, I'd get:
```
Error: error loading plugins: loading plugin /Users/ismail/.ipfs/plugins/lazyledger-plugin.so: plugin.Open("/Users/ismail/.ipfs/plugins/lazyledger-plugin"): plugin was built with a different version of package internal/unsafeheader
```